src/pages/_app.js

```diff
@@ -13,11 +14,13 @@ function MyApp({ Component, pageProps }) {
     }, []);
     return (
         <ThemeProvider theme={theme}>
-            <UploadTrackingProvider>
-                <CyverseAppBar>
-                    <Component {...pageProps} />
-                </CyverseAppBar>
-            </UploadTrackingProvider>
+            <UserProfileProvider>
```

Comment:
It looks like we need this Provider wrapper added as a Storybook decorator, similar to this:
https://github.com/cyverse-de/sonora/blob/e5d2ea1f3030395abb41a47ac787763c3a2b32bd/.storybook/config.js#L7-L9

Currently the AppBar story is throwing an error: `useUserProfile must be used within a UserProfileProvider`

Reply:
Ah, yes we do. Thanks for catching this. I'll get that fixed sometime today.